lib/polo/sql_translator.rb

```diff
@@ -72,7 +72,7 @@ def insert_values(record)
     module ActiveRecordFour
       def insert_values(record)
         connection = ActiveRecord::Base.connection
-        values = record.send(:arel_attributes_with_values_for_create, record.attribute_names)
+        values = record.send(:arel_attributes_with_values_for_create, connection.schema_cache.columns(record.class.table_name).map(&:name))
```

Comment:
Is there a reason to not use `record.class.column_names` to maintain parity with the `ActiveRecordFive.raw_sql` implementation?

Reply:
For whatever reason this method behaves differently on activerecord 4 compared to activerecord 5. On 4, `record.class.column_names` will include the virtual attribute.